integration_test/test_users.py

```diff
@@ -180,19 +180,15 @@ def add_admin_invalid_admin(self):
     def remove_user_valid_admin(self):
         admin = self.admin_users[0]
         fingerprint = "C92FE5A3FBD58DD3EC5AA26BB10116B8193F2DBD"
-        payload = {
-            "fingerprint": fingerprint,
-        }
         sig = self.admin_gpg.sign(
-            json.dumps(payload, sort_keys=True),
+            "NOCONTENT",
```

Comment:
- What is this string for?
  - Why aren't you e.g. just signing the empty string?
  - Doesn't this require client changes?

Reply:
Everything in this PR requires client changes.

When trying to sign empty content gpg was complaining. I had considered dropping the signing of content all together and moving to a more JWT like layout where the with header would have a base64 encoded string of the signing fingerprint along with a base64 encoded detached signature but didn't want to make more changes

Reply:
The string exists to have some piece of data to sign for the request. On the server we check the signature as a form of auth. To prevent replays we only allow something signed within a few seconds of the request